app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -477,6 +484,35 @@ def parse_stack_resource(resource, group)
         return uid, new_result
       end
 
+      def parse_managed_image(image)
+        uid = image.id.downcase
+
+        resource_group = ResourceGroup.where(:name => image.resource_group.downcase).find do |rg|
```

Comment:
@bronaghs I think I did that because I thought I might need it in the future for provisioning. But at the moment, no, we don't use it explicitly anywhere.